content/source/docs/plugin/framework/validation.html.md

```diff
@@ -0,0 +1,261 @@
+---
+layout: "extend"
+page_title: "Plugin Development - Framework: Validation"
+description: |-
+  How to validate configuration values using the provider development framework.
+---
+
+# Validation
+
+Practitioners implementing Terraform configurations desire feedback surrounding the syntax, types, and acceptable values. This feedback, typically referred to as validation, is preferably given as early as possible before a configuration is applied. Terraform supports validation for values in provider, resource, and data source configurations. The framework supports returning [diagnostics](./diagnostics.html) feedback on all of these, offering the ability to specify validations on an attribute, provider-defined type, and the entire provider, resource, or data source schema.
+
+~> **NOTE:** When implementing validation logic, configuration values may be [unknown](./types.html#unknown) based on the source of the value. Implementations must account for this case, typically by returning early without returning new diagnostics.
+
+## Syntax and Basic Schema Validation
```

Comment:
I want this heading to make the purpose clear - this is about when the checks you write with the framework occur, right? 

Also - just to be clear - if you write validation it will happen every single time we run one of the command you list below, right? So you write it one time and it gets applied everywhere.

Reply:
Newer practitioners and provider developers are generally curious about which type of validation is handled by which part of Terraform (CLI does some, SDK does some, provider does some). The general idea behind this section at a high level was to delineate between which of the validation responsibilities occurs where and when. I was thinking the general context might be helpful. It was maybe named poorly to highlight that this section is specifically what happens with Terraform CLI.

The Terraform CLI validation occurs before any framework/provider validation. Providers are not required to implement anything to receive the syntax and basic types validation. Maybe that's the key part missing from here that could help clarify this better.

Reply:
yeah I trust you on this but I wonder if we should say idk "Built-In Validation" "Default Validation"  or something similar? or even just "Default Terraform CLI Validation"?  yay. titles :p 

I just think this section title feels vague and so I want to be more specific about what it's about

Reply:
Oh! I like Default Terraform CLI Validation -- let's give that a go. Thank you!

```suggestion
## Default Terraform CLI Validation
```

We can circle back about this after the ❄️ , if desired. 😄